hw_diag/utilities/dbus_proxy/dbus_object.py

```diff
@@ -0,0 +1,21 @@
+import dbus
+
+from hw_diag.utilities.dbus_proxy.dbus_ids import DBusIds
+
+
+class DBusObject(object):
+    '''Base class for various dbus objects'''
+
+    def __init__(self, service, path, interface):
```

Comment:
We are trying to use type hints in new code FYI.

Reply:
added those.